exercises/diffie-hellman/.meta/hints.md

```diff
@@ -0,0 +1,18 @@
+# Hints
```

Comment:
Heading level 1 here is going to be a the wrong level for the README - the first level in this document should be level 2 (`##`). Personally, I would just delete this heading and keep the one below - two headings for one thing would seem strange in the flow of the README.